[docs] Add alert about disruption approval annotation (#19504)
deckhouse/deckhouse: docs/site/pages/virtualization-platform/documentation/admin/platform-management/platform-scaling/node/MIGRATING.md

## How to enable containerd v2

{% alert level="info" %}
During migration the directory `/var/lib/containerd` will be cleared, causing all pod images to be re-pulled, and the node will reboot.

> Also note that this is a [disruptive update](./node-management.html#disruptive-updates). If `approvalMode: Manual` is set, you need to run the update on the respective nodes with the annotation: `d8 k annotate node <NODE-NAME> update.node.deckhouse.io/disruption-approved=`.

Reviewer: ```suggestion
Also note that this is a [disruptive update](./node-management.html#disruptive-updates). If [`spec.disruptions.approvalMode: Manual`](/modules/node-manager/cr.html#nodegroup-v1-spec-disruptions-approvalmode) is set for a node group, run the update on the respective nodes with the annotation: `d8 k annotate node <NODE-NAME> update.node.deckhouse.io/disruption-approved=`.
```

   defaultCRI: ContainerdV2
   ```

> 2. **For a specific node group**. Set `ContainerdV2` in the [`spec.cri.type`](/modules/node-manager/cr.html#nodegroup-v1-spec-cri-type) parameter of the [NodeGroup](/modules/node-manager/cr.html#nodegroup) object.

Reviewer 2: ```suggestion
1. **For a specific node group**. Set `ContainerdV2` in the [`spec.cri.type`](/modules/node-manager/cr.html#nodegroup-v1-spec-cri-type) parameter of the [NodeGroup](/modules/node-manager/cr.html#nodegroup) object.
```
